### ISSUE TITLE
Release Apache ORC 1.7.5

### ISSUE THREAD

**Issue by dongjoon-hyun** [apache/orc#1155]
- [x] `branch-1.7` is healthy in GitHub Action
  - https://github.com/apache/orc/commits/branch-1.7
  - `Ubuntu 20.04`, `MacOS 10.15.7`, `MacOS 11.6.5`, `Windows` are tested.
  - `Java 8`, `Java 11`, `Java 17`, `Java 18` are tested.
  - `Clang 11.0.0` and `g++`
- [x] Docker tests (`CentOS 7`,  `Debian 10`, `Debian 11`, `Ubuntu 18`, `Ubuntu 20`, `Ubuntu 22`) passed.
- [x] Apache Spark master integration test passed.
  - https://github.com/dongjoon-hyun/spark/pull/7
- [x] Apache Iceberg master integration test passed.
  - https://github.com/williamhyun/iceberg/pull/2
- [x] Tag is created.
  - https://github.com/apache/orc/releases/tag/v1.7.5-rc0
- [x] Upload source artifact
  - https://dist.apache.org/repos/dist/dev/orc/v1.7.5-rc0
- [x] Publish java artifact
  - https://repository.apache.org/content/repositories/orgapacheorc-1058
- [x] Vote started.

**Comment by liujiawinds** [comment]
@dongjoon-hyun I have another pull request in progress about extracting the code for the encryption setup logic.

**Comment by dongjoon-hyun** [comment]
So?

**Comment by dongjoon-hyun** [comment]
In addition, FYI, this is a planed release according to the Apache ORC release cycle and Apache ORC release cycle is not blocked by un-opened PRs.

It's still unclear to me what you are intended here, but you can send to dev@orc if you want to share something.

**Comment by liujiawinds** [comment]
Sorry, I didn't explain clearly, I thought 1.7.5 would be released right after my last pr merged. 
Btw, I have submitted the pr mentioned above. #1156

**Comment by dongjoon-hyun** [comment]
I commented here, @liujiawinds .
- https://github.com/apache/orc/pull/1156#issuecomment-1152839362
